kvdb/postgres/db_conn_set.go

```diff
@@ -19,7 +19,8 @@ type dbConnSet struct {
 	dbConn         map[string]*dbConn
 	maxConnections int
 
-	sync.Mutex
+	// mu is used to guard access to the dbConn map.
+	mu sync.Mutex
```

Comment:
un-embedded this interface just to make it more clear that it is for the use of `dbConnSet` itself for protecting access to its dbConn member rather than giving the idea that it for the use of a user of `dbConnSet` (in other-words, callers should not be tempted to call `dbConnSet.Lock()` before using `dbConnSet` since this will cause deadlocks